pkg/skuba/actions/cluster/upgrade/plan.go

```diff
@@ -86,9 +86,17 @@ func plan(client clientset.Interface, availableVersions []*version.Version, clus
 			if nodeVersionInfo.EqualsClusterVersion(currentClusterVersion) {
 				fmt.Printf("  - %s: up to date", nodeName)
 			} else if !nodeVersionInfo.ToleratesClusterVersion(currentClusterVersion) {
-				fmt.Printf("  - %s; current version: %s (upgrade required)", nodeName, nodeVersionInfo.String())
+				if nodeVersionInfo.IsControlPlane() {
+					fmt.Printf("  - %s; current kubelet package version: %s, kube-apiserver container version: %s (upgrade required)", nodeName, nodeVersionInfo.KubeletVersion.String(), nodeVersionInfo.APIServerVersion.String())
```

Comment:
current kubelet **package** version is highly confusing, as kubeletversion is the version reported by kubelet, which is a runtime value. It means the package can technically be upgraded, as long as the process is not restarted, it will report the old version.

In the case of kubelet and an upgrade, the kubelet package _must be upgraded_ at some point and then restarted during the process.

I think it would be best to name this "Current reported kubelet version", but this is a nit. As this is a backport, I am fine with keeping this, but it tickles me a little bit.

Reply:
I created a new issue to tackle this.

Reply:
To be consistent we should report is as `current running kubelet version`